int8 STATE_ERROR=0
int8 STATE_OFF=1
int8 STATE_IDLE=2
int8 STATE_STANDBY=3
int8 STATE_ACTIVE=4

Header header
int8 state
bool ready # ready to transition to ACTIVE?
string info